specific_use_cases/DFU_work_in_progress/INTRO_annual_consumption_model.ipynb

```diff
@@ -129,253 +92,192 @@
-    "We'll use this to find the demand forecast zone that contains the weather station, then find the overlapping grid cells over which to compute the median value. The geometries of each demand forecast zone is available in our data catalog. You can grab the data as a pandas DataFrame object using the code provided below, or subset by forecast zones easily in `app.select` in the location subsetting tab. "
+    "#### 2b) Load the data into memory\n",
+    "\n",
+    "This may take some time, because the gridded data has to be loaded into memory. All computations we've done before this step are actually computed in this step; before, we just see a preview of the data. Because of this, **we recommend running this notebook in the Analytics Engine's Jupyter Hub, which provides additional computational resources that greatly speed up this step.**"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "ae93ca47-4c98-4867-a39c-22e9f6e264d5",
+   "id": "273d3440-d7a2-44c0-b3d0-f8ebfe4b0da9",
    "metadata": {},
    "outputs": [],
    "source": [
     "%%time\n",
-    "dfzs_df = Boundaries()._ca_forecast_zones # Load geometries from catalog\n",
-    "dfzs_df.head() # Display the first few rows  "
+    "data_dfz = app.load(data_dfz)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "8907cd0d-db63-4ae3-98a2-abd794f32c28",
-   "metadata": {},
-   "source": [
-    "### 3b) Crop the data to the corresponding forecast zone\n",
-    "We'll use [geopanda's `.contains` function](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.contains.html) to find the demand forecast zone where the weather station is located, and print the result. Then, we'll use [rioxarray](https://corteva.github.io/rioxarray/stable/rioxarray.html#rioxarray.raster_array.RasterArray.clip) to clip the data to the geometry that defines the forecast zone. "
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "40bc2023-4e3b-4ec8-9533-a1e3cc349cbb",
+   "id": "769cdbba-bc22-46ec-b610-e07aed14e3ec",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "data_dfz = clip_data_to_dfz(\n",
-    "    gridded_data=data, \n",
-    "    dfzs_df=dfzs_df, \n",
-    "    station_lat=one_station.LAT_Y,\n",
-    "    station_lon=one_station.LON_X\n",
-    ")"
+    "## Step 3: Compute the median value of the grid cells in station's corresponding forecast zone\n",
+    "\n",
+    "In this example, we will visuzalize the data across the Demand Forecast Zone for the Sacramento Municipal Utility District, and then calculate the median of all conditions across the Sacramento Municipal Utility District."
    ]
   },
   {
    "cell_type": "markdown",
    "id": "0bffd2d1-104a-4dbd-a308-1ed4efdee246",
    "metadata": {},
    "source": [
-    "### 3c) Visualize both the Demand Forecast Zone and the weather station on the same map "
+    "### 3a) Visualize both the Demand Forecast Zone and the weather station on the same map "
```

Comment:
Doesn't recognize 'one_station' for this location plot -- you could put back something reading that in, but just specify Sacramento instead of searching for the station in the zone. Alternatively you could direct the user to view this in app.select, although they wouldn't get to appreciate the data grid-spacing that way.